index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="./css/styles.css">
+    <link rel="stylesheet" type="text/css" href="./css/projects.css">
+    <link rel="stylesheet" type="text/css" href="./css/about.css">
+    <title>Profolio</title>
+</head>
+
+<body>
+
+    <section class="container">
```

Comment:
As requested by the previous reviewer, kindly remove this section tag here. The structure of your code should be:
`<html>`
`<body>`
inside body:
      `<header>`
        `<main>`
          `<footer>`

The `header` tag does not need to be in a section tag. Just have it as a `header`
It is important to follow hierarchy and use the correct semantic tags so as not to confuse screen readers.

Reply:
The Previous reviewer said I should use a Section tag as shown above. You are saying I should remove the section tag. I am repeating a week just because these reviews conflict with reviews from different reviewers.